### PR TITLE
Devise

### PR DESCRIPTION
**Pull request by katerspotaters**
I set up devise and added sign up and sign in. 
This is for "checkpoint 3"

### CODE REVIEW

File: spec/factories/models.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
```
Comment (brittanyjspringer) [inline spec/factories/models.rb]: @katerspotaters this needs to be removed, please. Just commit to the same branch.

File: spec/models/model_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (brittanyjspringer) [inline spec/models/model_spec.rb]: Remove please!

**Comment by brittanyjspringer** [comment]
@katerspotaters I left some notes to remove the "MODEL" tests from the repo. Please just commit the removals to the same branch as what you are PRing from.

**Comment by katerspotaters** [comment]
All removed and committed, just not pushed... should I do that?

**Comment by brittanyjspringer** [comment]
Yup! Push to the same branch, not Master.

**Comment by katerspotaters** [comment]
Okay cool all done!

**Pull request merged**